inst/qml/auditBayesianPlanning.qml

```diff
@@ -249,28 +247,44 @@ Form
 	{
 		title: 									qsTr("Probability Distribution")
 		name: 									"likelihood"
+		columns: 2
 
 		RadioButton
 		{
-			id: 								poisson
-			text: 								qsTr("Gamma")
-			name: 								"poisson"
-			checked: 							true
+			id: 								hypergeometric
+			text: 								qsTr("Beta-binomial")
+			name: 								"hypergeometric"
+			enabled:							n_units.value > 0
+		}
+
+		HelpButton
+		{
+			toolTip: 							qsTr("This distribution assumes you are working with a finite population.")
 		}
```

Comment:
I am not convinced of this use of the HelpButton... The resulting menu looks like this:
![Screenshot 2022-01-12 at 15 00 51](https://user-images.githubusercontent.com/7093546/149154516-d40351c8-3f3f-44f9-963e-3d14699912ea.png)

but clicking on any of the icons directs you to the main help file of the entire analysis. I would expect that each icon would lead to different files or at least scroll me to an appropriate location in the help file. 

I understand that you use it to display the tooltips, but perhaps instead we could just implement mouseover tooltips for qml elements? That seems more appropriate for this use-case...

Reply:
Agreed. For now I will remove this, but I will make an issue for a mouseover tooltip to be implemented.